Site Admin: Fix ESLint errors

tyxla opened the pull request:
## Proposed Changes

Fixes a few ESLint errors that were recently introduced with the `@automattic/site-admin` package.

## Why are these changes being made?

Just ESLint fixes.

## Testing Instructions

All checks should pass, specifically the Code Style check.

tyxla committed:
Site Admin: Fix ESLint errors

retrofox commented:
Thanks for addressing these issues, @tyxla. wondering why I didn't see them on time.

tyxla commented:
> Thanks for addressing these issues, @tyxla. wondering why I didn't see them on time.

Good question, it prompted me to do a research, and it seems like we have a flaw in our "Code Style" check.

I've documented it in an issue and will be happy to pick it up unless someone beats me to it:

- #99638

tyxla commented:
Got a fix for it here: #99729 and I'm using #99728 to demo that it works as expected.
